Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -274,8 +314,11 @@ void WebXROpaqueFramebuffer::resolveMSAAFramebuffer(GraphicsContextGL& gl)
     ASSERT(gl.checkFramebufferStatus(GL::READ_FRAMEBUFFER) == GL::FRAMEBUFFER_COMPLETE);
     gl.bindFramebuffer(GL::DRAW_FRAMEBUFFER, drawFBO);
     ASSERT(gl.checkFramebufferStatus(GL::DRAW_FRAMEBUFFER) == GL::FRAMEBUFFER_COMPLETE);
+#if USE(OPENXR)
     ScopedDisableScissorTest disableScissorTest { m_context };
+#endif
```

Comment:
This might have the same effect as the code at Line 241, but only for MSAA framebuffers.